mono/mini/mini-wasm-debugger.c

```diff
@@ -662,6 +674,49 @@ static gboolean describe_value(MonoType * type, gpointer addr)
 			}
 			break;
 		}
+		case MONO_TYPE_VALUETYPE:
+		case MONO_TYPE_TYPEDBYREF: {
+			MonoClass *klass = mono_class_from_mono_type_internal (type);
+			if (klass == mono_class_try_get_datetime_class ()) {
```

Comment:
It looks like these 3 blocks could be reduced to one that gets the the name from the klass? or at least reduce the duplication

Reply:
In the works

Reply:
Addressed in recent commits.